package.json

```diff
@@ -12,10 +12,9 @@
     "react-dom": "16.13.1",
     "react-measure": "2.2.2",
     "react-query": "^3.17.0",
-    "react-tooltip": "3.11.2",
```

Comment:
In SRC we're using React Tooltip 4, which doesn't seem to have a distributed UMD build (unlike this version). It's now bundled with SRC, so removing it here.